static/inline/js/libs/player.js

```diff
@@ -0,0 +1,295 @@
+        this.playButtonBorderRadius = isset(options.play_button_border_radius) ? options.play_button_border_radius : null;
+        this.playButtonBorderWidth = isset(options.play_button_border_width) ? options.play_button_border_width : null;
+        this.playButtonBorderColor = isset(options.play_button_border_color) ? options.play_button_border_color : null;
+        this.playButtonIconColor = isset(options.play_button_icon_color) ? options.play_button_icon_color : null;
+        this.playButtonWidth = isset(options.play_button_width) ? options.play_button_width : null;
+        this.playButtonHeight = isset(options.play_button_height) ? options.play_button_height : null;
+
+        this.onResize = isset(options.onResize) && isFunction(options.onResize) ? options.onResize : null;
+        this.onNext = isset(options.onNext) && isFunction(options.onNext) ? options.onNext : null;
+
+        this.instance = null;
+        this.el = 'string' === typeof el ? document.getElementById(el) : el;
+
+        this.assets = (function(data){
+            var assets = [];
+            for (var i = 0; i < data.length; i++) {
+                var video = data[i];
+
+                if (isEmpty(video)) break;
+
+                var asset = video.asset;
+                asset.assetId = video.id;
+                asset.assetTitle = video.title;
+                asset.loginId = video.loginId;
+
+                if (isset(video,'events') && video.events.length) {
+                    asset.analyticsLogUrl = video.analytics;
+                    asset.analyticsObj = video.events[1].data;
+                } else {
+                    asset.analyticsObj = {
+                        pg: isset(video,'parentId') ? video.parentId : ( isset(options,'channel') ? options.channel.id : 0 ),
+                        vd: video.id,
+                        li: video.loginId
+                    };
+                }
+
+                if (!asset.sources) asset.sources = [{ file: asset.videoId }];
+                asset.type = asset.type || 'youtube';
+                assets.push(asset);
+            }
+            return assets;
+        }(options.data));
+
+
+        //Context reference for Methods.
+        var that = this;
+
+        //Sometimes we want/need to show an intearctive overlay on top of the player. We need this for MP4 videos that will
+        //cue (mobile or autoplay:off) to actual play the video on demand.
+        this.addOverlay = function(asset){
+            var overlay = document.createElement('div');
+            overlay.classList.add('tvp-overlay');
+            overlay.style.backgroundImage = 'url("' + asset.thumbnailUrl + '")';
+
+            var overlayColor = this.overlayColor ? '#' + this.overlayColor : 'transparent';
+            var overlayHtml = '<div class="tvp-overlay-cover" style="opacity:' + this.overlayOpacity + ';' +
```

Comment:
All HTML should be a partial file that is loaded into the _bootstrap.html_

Please refer to _Carousel_ example:

https://github.com/tvpage/tvpage-hugo-template/tree/master/layouts/partials/carousel
https://github.com/tvpage/tvpage-hugo-template/blob/master/layouts/carousel/bootstrap.html